CHANGELOG.md

```diff
@@ -136,6 +138,7 @@
 ## stream-chat-android-ui-common
 ### ✅ Added
 - Exposed a way to allow you to include the current user avatar in the Channel avatar [#4561](https://github.com/GetStream/stream-chat-android/pull/4561)
+- `MessageComposerController` will now query the server for a list of channel members if the input contains a mention symbol (@) and no user name matching the expression after the symbol @ was found in the local state containing a list of channel members. [#4647](https://github.com/GetStream/stream-chat-android/pull/4647)
```

Comment:
Added here by accident, actually belong in the unreleased segment as this was ultimately released in v5.12.0.

Will be updated to reflect the correct state in the relevant branches (`v5`, `main`, `develop`) , but also leaving a comment here for posterity should anyone find themselves looking at the PR and wondering.